AmericanMedicalAssociation/joe-style-guide
# EWLJ-714: Update Utility Menu Twitter Logo

## JIRA Ticket(s)

- See ticket [EWLJ-714 Update Twitter logo on utility menu](https://palantir.atlassian.net/browse/EWLJ-714).

## Description

Updates twitter logo

## Testing instructions

- visit an article such as http://journalofethics.lndo.site/article/why-should-we-care-about-mental-health-older-adults/2023-10
- observe the twitter logo is now an X

## Relevant Screenshots/gifs:
<img width="1873" alt="Screenshot 2023-11-03 at 3 00 33 PM" src="https://github.com/AmericanMedicalAssociation/code-of-ethics/assets/2271747/15155fff-f91b-4bf5-b0a1-62e1a8ce3e20">



## Remaining tasks

_Delete this section if there are no remaining tasks._


## Additional Notes:

_Anything more to add?_


---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
